governance/polls/WBTC Risk Premium Adjustment - August 24, 2020

```diff
@@ -0,0 +1,42 @@
+---
+title: WBTC Risk Premium Adjustment - August 24, 2020
+summary: Signal your support to adjust the WBTC Risk Prmium within a range of 0% to 6%
+discussion_link: https://forum.makerdao.com/t/2406
+poll_rules: The voter may select to vote for one of the poll options or they may elect to abstain from the poll entirely
+options:
```

Comment:
This should be a yes/no poll based on the highest signal result, which I believe was 4%.